Source/WebKit/UIProcess/API/Cocoa/WKWebpagePreferences.mm

```diff
@@ -640,6 +646,12 @@ - (void)_setNetworkConnectionIntegrityPolicy:(_WKWebsiteNetworkConnectionIntegri
     if (networkConnectionIntegrityPolicy & _WKWebsiteNetworkConnectionIntegrityPolicyFailClosed)
         webCorePolicy.add(WebCore::NetworkConnectionIntegrity::FailClosed);
 
+    if (networkConnectionIntegrityPolicy & _WKWebsiteNetworkConnectionIntegrityPolicyWebSearchContent)
```

Comment:
would it be better to extract the `networkConnectionIntegrityPolicy` condition into an outer `if` since all of these if conditions use it?

Reply:
> would it be better to extract the `networkConnectionIntegrityPolicy` condition into an outer `if` since all of these if conditions use it?

Could you elaborate? (i.e. what would the outer if condition be?)

Reply:
oh whoops I misread `&` as `&&`